patronload/config.py

```diff
@@ -24,10 +38,32 @@ def configure_logger(logger: logging.Logger, verbose: bool) -> str:
     )
 
 
-def configure_sentry() -> str:
-    env = os.getenv("WORKSPACE")
-    sentry_dsn = os.getenv("SENTRY_DSN")
+def configure_sentry(env: str, sentry_dsn: str) -> str:
     if sentry_dsn and sentry_dsn.lower() != "none":
         sentry_sdk.init(sentry_dsn, environment=env)
         return f"Sentry DSN found, exceptions will be sent to Sentry with env={env}"
     return "No Sentry DSN found, exceptions will not be sent to Sentry"
+
+
+def load_config_values() -> dict:
+    """
+    Retrieve all required env variables and return as a dict.
+
+    If an env variable is not present, a KeyError will be raised by the
+    get_required_env_variable function with a reminder that it is required
+    by the application.
+    """
+    config_values = {}
+    for expected_config_variable in EXPECTED_CONFIG_VARIABLES:
+        if config_value := get_required_env_variable(expected_config_variable):
+            config_values[expected_config_variable] = config_value
+    return config_values
+
+
+def get_required_env_variable(variable_name: str) -> Optional[str]:
+    try:
+        return os.environ[variable_name]
+    except KeyError as error:
+        raise KeyError(
+            f"Missing variable: '{variable_name}' is required for all environments"
```

Comment:
We explicitly state in the README that not all of the variables are required in all envs, so we either need to update that or change the requirement here. I would suggest the latter, as they really aren't all required in all envs.